doc/introduction/compiling_circuits.rst

```diff
@@ -274,6 +274,14 @@ enabled.
 Default behaviour with custom decompositions
 ********************************************
 
+.. warning::
+    The keyword argument for defining custom quantum gate decompositions, ``custom_decomps``,
+    has been deprecated and will be removed in v0.44. Instead, to specify custom decompositions for
```

Comment:
Right now we are developing v0.44 right? Is this ``custom_decomps`` already deprecated and we are removing it somewhere?